app/views/about/index.html.erb

```diff
@@ -1,122 +1,13 @@
-                    <td><div class="label label-success"><%= _('Installed') %></div></td>
-                  <% else %>
-                    <td><div class="label label-default"><%= _('Not Installed') %></div></td>
-                  <% end %>
-                </tr>
-              <% end %>
-              </tbody>
-            </table>
-            <%= _('To enable a provider, either install the OS package (e.g. foreman-libvirt) or enable the bundler group for development setup (e.g. ovirt).') %>
-          </div>
-          <div class="tab-pane" id="compute_resources">
-            <% if @compute_resources.empty? %>
-              <p class="ca"><%= _("No compute resource to show") %></p>
-            <% else %>
-              <table class="table table-striped table-fixed">
-                <thead>
-                  <tr>
-                    <th class="col-md-4"><%= _("Name") %></th>
-                    <th class="col-md-4"><%= _("Type") %></th>
-                    <th class="col-md-4"><%= _("Status") %></th>
-                  </tr>
-                </thead>
-                <tbody>
-                <% @compute_resources.each do |compute| %>
-                  <tr>
-                    <td class="ellipsis"><%= link_to(compute.name, compute) %></td>
-                    <td><%= compute.provider_friendly_name %></td>
-                    <td><div class="compute-status" data-url=<%= ping_compute_resource_path(compute) %>><%= spinner %></div></td>
-                  </tr>
-                <% end %>
-                </tbody>
-              </table>
-            <% end %>
-          </div>
-        <% end %>
-        <div class="tab-pane" id="plugins">
-          <% if @plugins.empty? %>
-            <p class="ca"><%= _("No plugins found") %></p>
-          <% else %>
-            <table class="table table-striped table-fixed">
-              <thead>
-                <tr>
-                  <th class="col-md-4"><%= _("Name") %></th>
-                  <th class="col-md-6"><%= _("Description") %></th>
-                  <th class="col-md-2"><%= _("Author") %></th>
-                  <th class="col-md-2"><%= _("Version") %></th>
-                </tr>
-              </thead>
-              <tbody>
-              <% @plugins.each do |plugin| %>
-                <tr>
-                  <td><%= plugin.url.blank? ? plugin.name : link_to(plugin.name, plugin.url, :rel=>'external')%></td>
-                  <td class="ellipsis"><%= _(plugin.description) %></td>
-                  <td class="ellipsis"><%= plugin.author_url.blank? ? plugin.author : link_to(plugin.author, plugin.author_url)%></td>
-                  <td><%= plugin.version %></td>
-                </tr>
-              <% end %>
-              </tbody>
-            </table>
-          <% end %>
-        </div>
-      </div>
+      <div id="tabs"></div>   
+      <%= mount_react_component('About', '#tabs', {:compute => compute_resources, :proxy => proxies,
```

Comment:
probably makes sense to combine it under a method, so in theory we can expand it later vs hard code it. e.g.

```ruby
def about_data
  {:compute => compute_resources, :proxy => proxies,...
end
```